drivers/hmis_csv_importer/app/views/hmis_csv_importer/import_overrides/new.haml

```diff
@@ -10,6 +10,7 @@
       .well
         .form-inputs
           = f.input :file_name, collection: HmisCsvImporter::ImportOverride.available_classes, label: 'File', as: :select_two, include_blank: false
+          = f.input :description, label: 'Description', hint: 'Why is this override being created?', required: false
```

Comment:
required is false by default.
```suggestion
          = f.input :description, label: 'Description', hint: 'Why is this override being created?'
```

Reply:
This driver appears to be using the opposite logic and defaulting to true. The `required: false` tag is being applied to the optional fields in the views.

Reply:
weird.  Thanks